lighthouse-core/report/report-generator.js

```diff
@@ -151,8 +151,10 @@ class ReportGenerator {
       }
 
       const partials = reportPartials.report.partials;
-      Handlebars.registerPartial(audit.name,
-        Handlebars.template(partials[audit.name] || partials['null-formatter']));
+      const partial = partials[audit.name] ||
+          partials[audit.extendedInfo.formatter] ||
```

Comment:
Yeah we have a mismatch of kebab-case and camelCase here, so better to address that then keep the extra `||`.

@sendilkumarn no worries, `critical-request-chains` worked because the name of the audit is the same as the formatter :)